api/citations/utils.py

```diff
@@ -27,4 +27,15 @@ def warn(citation_item):
 
     bibliography.cite(citation, warn)
     bib = bibliography.bibliography()
-    return unicode(bib[0] if len(bib) else '')
+
+    if len(bib):
+        if style == 'apa':
```

Comment:
Could you put in tests for these cases? I'd like to make sure that if someone fixes CiteProc that what you have here won't go weird.